pkg/proxy/iptables/proxier_test.go

```diff
@@ -2506,4 +2506,61 @@ COMMIT
 	assert.NotEqual(t, expectedIPTablesWithSlice, fp.iptablesData.String())
 }
 
+func TestLoadBalancerIngressRouteTypeProxy(t *testing.T) {
+	ipt := iptablestest.NewFake()
+	fp := NewFakeProxier(ipt, false)
+	svcIP := "10.20.30.41"
```

Comment:
table test as in https://github.com/kubernetes/kubernetes/pull/92312/files/0ad979b5324215cdf50bdb4714173b059490a4f8#r443197212 but using `expectedRule bool` instead of `expectedServices    int`?